firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -78,7 +78,15 @@
   private static final String CHIME_FIREBASE_APP_NAME = "CHIME_ANDROID_SDK";
 
   private static final ThreadFactory threadFactory =
-      new ThreadFactoryBuilder().setNameFormat("firebase-installations-executor-%d").build();
+      new ThreadFactory() {
+        private final AtomicInteger mCount = new AtomicInteger(1);
+
+        @Override
+        public Thread newThread(Runnable r) {
+          return new Thread(
+              r, String.format("firebase-installations-executor-%d", mCount.getAndIncrement()));
```

Comment:
Can we make this uppercase this time?
I know that would mean I have to submit a new CL in Google3...
Ok, as discussed offline:
Let's not do it.
We can discuss after the successful release.